src/type_checker/resources/primitives/complex.py

```diff
@@ -0,0 +1,26 @@
+class complex:
+    def __add__(self, other: int) -> complex: pass
+    def __add__(self, other: float) -> complex: pass
+    def __add__(self, other: complex) -> complex: pass
+    def __add__(self): return self
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The special method '__add__' expects 1 param(s), 0 was given](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4163821)